packages/office-ui-fabric-react/src/components/__snapshots__/Checkbox.Other.Example.tsx.shot

```diff
@@ -1570,11 +1559,7 @@ exports[`Component Examples renders Checkbox.Other.Example.tsx correctly 1`] = `
               height: 10px;
               left: 4px;
               position: absolute;
-              right: 0px;
               top: 4px;
-              transition-duration: 200ms;
-              transition-property: border-width, border, border-color;
-              transition-timing-function: cubic-bezier(.4, 0, .23, 1);
```

Comment:
Are these changes intentional? I had started on a PR myself, but I didn't htink there'd be any changes to actual styling in snapshots.

Reply:
My mistake. I commented them out looking for fat to trim and deleted them on accident before the push. I'll pop them back, thanks!